src/view-handler/index.js

```diff
@@ -55,6 +56,31 @@ export default function createViewHandler({ layoutService, extremumModel, model,
     setMeta(newMeta) {
       extend(true, meta, newMeta);
     },
+
+    setInteractionInProgress: (newState) => {
+      interactionInProgress = newState;
+    },
+    rendererSettings: {
```

Comment:
no problem. You were clear. I just over-optimised since both point and point label components have the same rect. I have moved `canvasBufferSize` back out.

Reply:
Now we have codeClimate complaint about code duplication  😄